### PR TITLE
[WIP] Ladder climbing speed property added Fixes #1563

### LINKED ISSUE #1563: Ladder speed
A user on the Discord (and another on the forum last year) said he wanted to change the ladder climbing speed.
Right now, the ladder climbing speed of the platformer behavior is fixed.
The only way to change it is changing the time scale, which is quite impractical in most situations.
It should be possible to change it like the moving speed and other parameters.

The related files are in GDevelop/Extensions/PlatformBehavior and the related lines are:
image
That 150 should be replaced by an attribute which should be declared in the file PlatformerObjectBehavior.cpp. Looks fairly easy... I think. 😁

### PR DESCRIPTION
**Pull request by ozata**
Fixes #1563

### CODE REVIEW

**Comment by ozata** [comment]
@4ian Could you review?

File: Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h [hunk 1]
```diff
@@ -184,6 +186,7 @@ class GD_EXTENSION_API PlatformerObjectRuntimeBehavior
   double deceleration;     ///< In pixels.seconds^-2
   double maxSpeed;         ///< In pixels.seconds^-1
   double jumpSpeed;        ///< In pixels.seconds^-1
+  double ladderClimbingSpeed;
```
Comment (4ian) [inline Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h]: Add a comment for consistency with other members.

File: Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h [hunk 2]
```diff
@@ -37,6 +37,7 @@ class GD_EXTENSION_API PlatformerObjectRuntimeBehavior
   double GetDeceleration() const { return deceleration; };
   double GetMaxSpeed() const { return maxSpeed; };
   double GetJumpSpeed() const { return jumpSpeed; };
+  double GetLadderClimbingSpeed() const {return ladderClimbingSpeed; };
```
Comment (4ian) [inline Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h]: Be consistent in your spacing

File: Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp [hunk 3]
```diff
@@ -39,6 +39,7 @@ void PlatformerObjectBehavior::InitializeContent(
   behaviorContent.SetAttribute("canGrabPlatforms", false);
   behaviorContent.SetAttribute("yGrabOffset", 0);
   behaviorContent.SetAttribute("xGrabTolerance", 10);
+  behaviorContent.SetAttribute("ladderClimbingSpeed",150);
```
Comment (4ian) [inline Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp]: nitpicking: Spacing

**Comment by ozata** [comment]
Please don't review this yet, it passed CI but I detected a bug with my implementation. Sorry for pushing it.

**Comment by arthuro555** [comment]
This or can be closed as the linked issue has already been worked on in #1578

**Pull request closed without merging**